data/abilities.js

```diff
@@ -2264,18 +2264,16 @@ let BattleAbilities = {
 	"parentalbond": {
 		desc: "This Pokemon's damaging moves become multi-hit moves that hit twice. The second hit has its damage quartered. Does not affect multi-hit moves or moves that have multiple targets.",
 		shortDesc: "This Pokemon's damaging moves hit twice. The second hit has its damage quartered.",
+		// second hit modification is in script.js under tryMoveHit to allow Mummy interaction to work
 		onPrepareHit: function (source, target, move) {
 			if (['iceball', 'rollout'].includes(move.id)) return;
 			if (move.category !== 'Status' && !move.selfdestruct && !move.multihit && !move.flags['charge'] && !move.spreadHit && !move.isZ) {
 				move.multihit = 2;
 				move.hasParentalBond = true;
+				move.basePowerModifier = 0.25;
```

Comment:
I was confused by this apparent reuse of `basePowerModifier`, but I don't actually see anyone else setting it, so maybe those uses are obsolete. Either way, you're now setting and testing two variables at the same time, so one of the checks seems to be unnecessary?

Reply:
I think you're right, I will try to change hasParentalBond into parentalBondModifier and just run a check on that.